src/components/Viewer/queries.js

```diff
@@ -14,3 +14,11 @@ export const buildViewerFileQuery = fileId => ({
     fetchPolicy: defaultFetchPolicy
   }
 })
+
+export const getContactByIds = ids => ({
+  definition: () => Q('io.cozy.contacts').getByIds(ids),
+  options: {
+    as: `getContactByIds${ids.map(id => id)}`,
```

Comment:
The naming system for a single get (getById) is : 
as: `doctype/id` > cf https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/store/queries.js#L511

wondering if we should not have one for getByIds():
as: `doctype/ids.join('')`

I think it worth it.